champions/src/main/java/me/mykindos/betterpvp/champions/champions/builds/RandomBuild.java

```diff
@@ -0,0 +1,57 @@
+package me.mykindos.betterpvp.champions.champions.builds;
+
+import com.google.inject.Singleton;
+import me.mykindos.betterpvp.champions.champions.skills.ChampionsSkillManager;
+import me.mykindos.betterpvp.champions.champions.skills.Skill;
+import me.mykindos.betterpvp.core.components.champions.Role;
+import me.mykindos.betterpvp.core.utilities.UtilMath;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+//Generate a random build
+@Singleton
+public class RandomBuild {
```

Comment:
This shouldn't be a static method, could just go into BuildManager or BuildRepository which can be accessed via BuildMenu

Reply:
I was originally trying to avoid putting ChampionsSkillManager in BuildManager. Fixed